features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/partials/edit-asset.hbs

```diff
@@ -22,8 +22,16 @@
         <div class="form-group">
             <label class="control-label col-sm-2">Owner :  </label>
             <div class="col-sm-10">
-                <input id='overview_businessOwner' name='overview_businessOwner' class="col-lg-6 col-sm-12 col-xs-12" type='text' value="{{businessOwnerName}}"  placeholder=" Owner Name"  />
+                <select class="selectOwnerName">
+                    <option></option>
+                    <option value={{businessOwner.businessOwnerId}}
+                            selected="selected">{{businessOwner.businessOwnerName}} ({{businessOwner.businessOwnerEmail}})</option>
+
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5